src/components/common/AppHeader.vue

```diff
@@ -159,7 +159,6 @@ export default {
   .app-header > .container {
     position: fixed;
     height: 100%;
-    overflow: scroll;
```

Comment:
this should be `overflow: auto`

i set it to `scroll` to fix the bug you reported about the menu not being visible on certain screen sizes. scroll forces scrollbars on windows and linux, but on mac os it only displays them if the content of the container is greater than the container (when scrollbars are actually needed).

Reply:
Thank you for the explanation! Maybe my solution was a bit bold :sweat_smile: